### PR TITLE
CNDIT-224: Testcontainers for MS SQL Server

### PR DESCRIPTION
**Pull request by bspuli**

### CODE REVIEW

File: report-service/src/main/java/gov/cdc/dataingestion/config/KafkaConfig.java [hunk 1]
```diff
@@ -18,13 +18,13 @@
 @EnableKafka
 @Configuration
 public class KafkaConfig {
-    @Value("${spring.kafka.group-id}")
+    @Value("${spring.kafka.group-id:}")
     private String groupId = "";
 
-    @Value("${spring.kafka.bootstrap-servers}")
+    @Value("${spring.kafka.bootstrap-servers:}")
```
Comment (sdeusch) [inline report-service/src/main/java/gov/cdc/dataingestion/config/KafkaConfig.java:24]: What are the : at the end of the variable name?

File: report-service/src/main/java/gov/cdc/dataingestion/rawmessage/service/RawELRService.java [hunk 2]
```diff
@@ -19,15 +19,15 @@
 public class RawELRService {
 
     private static final String CREATED_BY = "admin";
-    @Value("${kafka.raw.topic}")
-    String topicName;
+    //@Value("${kafka.raw.topic}")
```
Comment (sdeusch) [inline report-service/src/main/java/gov/cdc/dataingestion/rawmessage/service/RawELRService.java:22]: if this is dead code it should be removed not commented out.

File: report-service/src/test/java/gov/cdc/dataingestion/it/RawMessageServiceIT.java
Comment (sdeusch) [inline report-service/src/test/java/gov/cdc/dataingestion/it/RawMessageServiceIT.java:1]: Shouldn't a test class have *Test* in the name?

**Review by sdeusch — APPROVED** [review]

File: report-service/src/main/java/gov/cdc/dataingestion/config/KafkaConfig.java (hunk 1 above)
Comment (bspuli) [reply report-service/src/main/java/gov/cdc/dataingestion/config/KafkaConfig.java:24]: That is for default value. Here default value is empty string.

**Pull request merged**